scripts/distributed_scripts/distributed_configs/distributed_client_1.yaml

```diff
@@ -12,7 +12,7 @@ distribute:
   client_host: '127.0.0.1'
   client_port: 50052
   role: 'client'
-  data_file: 'toy_data/client_1_data'
+  data_idx: 1
```

Comment:
TODO: provide an example that loads data from separate data files in distributed mode